src/redux/sagas/projectSaga.tsx

```diff
@@ -1,22 +1,23 @@
 import axios from "axios";
 import { all, call, put, takeLatest } from "redux-saga/effects";
 
-import {  IPROJECT } from "../actions/types";
-import {  FETCH_ALLPROJECTS_REQUEST} from "../actions/actionTypes";
-import { fetchProjectFailure, fetchProjectSuccess } from "../actions/fetchProjectAction";
+import { IPROJECT, DeleteProjectInfoAction, DeleteProjectGroupsInfoAction } from "../actions/types";
+import { FETCH_ALLPROJECTS_REQUEST } from "../actions/actionTypes";
+import { fetchProjectFailure, fetchProjectSuccess, deleteProjectInfoActionSuccess, deleteProjectInfoActionFailure, deleteProjectGroupsInfoAction, deleteProjectGroupsInfoActionSuccess, deleteProjectGroupsInfoActionFailure } from "../actions/fetchProjectAction";
+import { DELETE_PROJECTS_INFO_ACTION } from "../actions/projectActionTypes";
 
 
 const getProjects = () =>
-axios.get<IPROJECT[]>("http://localhost:8080/sourcecourse/home");
+  axios.get<IPROJECT[]>("http://localhost:8080/sourcecourse/home");
```

Comment:
Are we keeping the base urls in individual files or in a constant

Reply:
we are not keeping base url in individual file